src/test/comment/textContent.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import test from 'ava';
+import { Comment } from '../../worker-thread/dom/Comment';
+
+test.beforeEach(t => {
+  t.context = {
+    comment: new Comment('default value'),
+  };
+});
+
+test('get textContent', t => {
+  const { comment } = t.context as { comment: Comment };
+
+  t.is(comment.textContent, 'default value');
+});
+
+test('set textContent', t => {
+  const { comment } = t.context as { comment: Comment };
+
```

Comment:
Add a test to assert that it is `default value` before
```suggestion

t.is(comment.textContent, 'default value');
```